### DISCUSSION TITLE
New AI RPI/Sony Camera

### DISCUSSION THREAD

**Discussion by WirthmU** [aaronwmorris/indi-allsky#1551] [General]
What can we expect from the new RPI camera with integrated AI that has just been anounced?

**Comment by aaronwmorris** [comment]
Link for reference:  https://www.raspberrypi.com/documentation/accessories/ai-camera.html

I was just reading about that camera last night.  That is very interesting.

@gordtulloch I wonder if this is something that could be used as the backend for [mlCloudDetect](https://github.com/gordtulloch/mlCloudDetect)

**Comment by gordtulloch** [comment]
Seems pretty interesting for a lot of applications! I wonder how big a model can fit on the module? I'll order one and try it out!
